RELEASE-NOTES.txt

```diff
@@ -1,3 +1,7 @@
+1.28.1
+------
+* [**] Avoid crash when editor state becomes invalid
```

Comment:
WDYT about adding a more specific reference? Something like: "...when editor's selection state becomes..." for instance.

Reply:
Yes, that's better. Updated in 8cc2b62!